test/test_source_gdal.py

```diff
@@ -580,3 +580,24 @@ def testNoData():
         imagePath, projection='EPSG:3857',
         style={'bands': [{'band': 1, 'max': 100, 'min': 5, 'nodata': 0}]})
     assert source.getThumbnail()[0]
+
+
+def testAlphaProjection():
+    testDir = os.path.dirname(os.path.realpath(__file__))
+    imagePath = os.path.join(testDir, 'test_files', 'rgba_geotiff.tiff')
+    source = large_image_source_gdal.open(
+        imagePath, projection='EPSG:3857')
+    base = source.getThumbnail(encoding='PNG')[0]
+    basenp = source.getThumbnail(format='numpy')[0]
+    assert numpy.count_nonzero(basenp[:, :, 3] == 255) > 30000
+    source = large_image_source_gdal.open(
+        imagePath, projection='EPSG:3857',
+        style={'bands': [
+            {'band': 1, 'palette': 'R'},
+            {'band': 2, 'palette': 'G'},
+            {'band': 3, 'palette': 'B'}]})
+    assert source.getThumbnail(encoding='PNG')[0] == base
+    assert not (source.getThumbnail(format='numpy')[0] - basenp).any()
+    source = large_image_source_gdal.open(
+        imagePath)
+    assert numpy.count_nonzero(source.getThumbnail(format='numpy')[0][:, :, 3] == 255) > 30000
```

Comment:
Awesome, thanks for adding this test. Was the bug in #912 repeatable with this test file, `'rgba_geotiff.tiff'`?

Reply:
Ah just saw your comment that you created that file for this test. Perfect!

Reply:
Yes.